packages/fxa-content-server/app/scripts/views/post_verify/cad_qr/get_started.js

```diff
@@ -8,21 +8,36 @@ import FlowEventsMixin from './../../mixins/flow-events-mixin';
 import FormView from '../../form';
 import Template from 'templates/post_verify/cad_qr/get_started.mustache';
 import preventDefaultThen from '../../decorators/prevent_default_then';
+import { MOZ_ORG_SYNC_GET_STARTED_LINK } from '../../../lib/constants';
 
 class GetStarted extends FormView {
   template = Template;
   viewName = 'get-started';
+  settingReminder = false;
 
   events = assign(this.events, {
-    'click #maybe-later-btn': preventDefaultThen('clickMaybeLater'),
+    'click #maybe-later-link': preventDefaultThen('clickMaybeLater'),
   });
 
   submit() {
     return this.navigate('/post_verify/cad_qr/ready_to_scan');
   }
 
   clickMaybeLater() {
-    return this.navigate('/sms');
+    if (this.settingReminder) {
+      return;
+    }
+
+    this.settingReminder = true;
+    const account = this.getSignedInAccount();
+    return account
+      .createCadReminder()
+      .then(() => {
+        this.navigateAway(MOZ_ORG_SYNC_GET_STARTED_LINK);
+      })
+      .catch(() => {
+        this.settingReminder = false;
```

Comment:
The likelihood of this catching is pretty low, but I wonder if this should `alert` or something minimal to inform of the error. I'm leaning towards no, but could be convinced otherwise.